app/javascript/packages/normalize-yaml/cli.js

```diff
@@ -9,11 +9,23 @@ const { readFile, writeFile } = fsPromises;
 /** @type {Record<string,any>=} */
 const prettierConfig = prettier.resolveConfig.sync(process.cwd());
 
-const files = process.argv.slice(2);
+const args = process.argv.slice(2);
+const files = args.filter((arg) => !arg.startsWith('-'));
+const flags = args.filter((arg) => arg.startsWith('-'));
+
+/** @type {import('./index').NormalizeOptions} */
+const options = {
+  prettierConfig,
+  exclude: /** @type {import('./index').Formatter[]} */ ([
+    flags.includes('--disable-sort-keys') && 'sortKeys',
+    flags.includes('--disable-format-content') && 'formatContent',
```

Comment:
what if we call it --disable-smart-punctuation?

Reply:
I like that. 👍  As soon as I pushed, I had a feeling "format content" was still falling victim to the same ambiguity as before 😅

Reply:
![:crying-cowboy-sunglasses:](https://emojis.slackmojis.com/emojis/images/1594839785/9702/crying-sunglasses-cowboy.png?1594839785)

Reply:
Alright, I think we're finally set now with 0006c35 and aff1d50,